docs/guide/integrations/multistore.md

```diff
@@ -204,3 +244,119 @@ or
   "
 ></router-link>
 ```
+
+
+## Useful _Helpers_
+
+### How to get current store view?
+
+Here is a helper method to get the value of current store view.
+
+You just need to import `currentStoreView` function from `core/lib/multistore` as example follows : 
+
+```js
+
+import { currentStoreView } from '@vue-storefront/core/lib/multistore'
+
+// ... abridged 
+
+
+  return amount: {
+                currency: currentStoreView().i18n.currencyCode,
+                value: method.price_incl_tax
+              }
+
+```
+
+`currentStoreView()` will return the object value you set in `local.json`
```

Comment:
I would show the shape of this object for readers:
```ts
interface StoreView {
  storeCode: string,
  extend?: string,
  disabled?: boolean,
  storeId: any,
  name?: string,
  url?: string,
  appendStoreCode?: boolean,
  elasticsearch: {
    host: string,
    index: string
  },
  tax: {
    sourcePriceIncludesTax?: boolean,
    finalPriceIncludesTax?: boolean,
    deprecatedPriceFieldsSupport?: boolean,
    defaultCountry: string,
    defaultRegion: null | string,
    calculateServerSide: boolean,
    userGroupId?: number,
    useOnlyDefaultUserGroupId: boolean
  },
  i18n: {
    fullCountryName: string,
    fullLanguageName: string,
    defaultLanguage: string,
    defaultCountry: string,
    defaultLocale: string,
    currencyCode: string,
    currencySign: string,
    currencyDecimal: string,
    currencyGroup: string,
    fractionDigits: number,
    priceFormat: string,
    dateFormat: string
  },
  seo: {
    defaultTitle: string
  }
}
```

Reply:
Very nice